api-spec/filterapi_v2_messages.yaml

```diff
@@ -1,7 +1,7 @@
 # /filter/v2/messages/{contentTopic}:
 get: # get_waku_v2_filter_v2_messages
   summary: Get the latest messages on the polled content topic
-  description: Get a list of messages that were received on a subscribed content topic after the last time this method was called.
+  description: Get a list of messages that were received on a subscribed content topic after the last time this method was called. Messages are selected from the last 50 messages received by the node.
```

Comment:
Unlike /relay REST API we have no config to influence filter cache... IDK why we do not... so 50 is the global default for sure.

Reply:
I would avoid mentioning a fixed value, i.e. 50, so that we don't need to update that in case we modify it in the code. This recommendation doesn't apply to the other suggested changes as it seems the user can change the `30` value for CLI argument

Reply:
Yes, I also thought the same! If we change the value we would need to update the docs. The issue is that I think it's important to have this value documented, otherwise the user will be in the dark on what to expect to receive, that's what happened to @AlbertoSoutullo and if we don't specify the value, I'm afraid others will get confused or misuse the endpoint

I don't think this is something that should be updated very often, I think it's better to have that burden than not documenting what to expect from the endpoint